app/controllers/concerns/pagination_data.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module PaginationData
+  extend ActiveSupport::Concern
+
+  def pagination_data(objects, default_page: nil, default_per_page: nil)
```

Comment:
I can't quite tell if we really need to keep this `default_page` argument or not. It feels like we could remove it and change `params[:page] || default_page` to `params[:page] || 1` below, and not bother passing in this argument elsewhere in the code? What do you think?